docs/sources/panels-visualizations/visualizations/table/index.md

```diff
@@ -122,16 +127,16 @@ If you set these in the Field tab, then the type will apply to all fields, inclu
 
 The **Auto** cell type automatically displays values, with sensible defaults applied.
 
-Toggle the **Wrap text** switch to wrap text in the cell with the longest content or the matched field, if you've set a field override.
+### Wrap text
```

Comment:
If we're gonna add a heading here, this needs to go down a level since it's not a call type, but an option within the Auto cell type.
```suggestion
#### Wrap text
```